applications/convection_diffusion_application/python_scripts/convection_diffusion_transient_solver.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Check that applications were imported in the main script
+KratosMultiphysics.CheckRegisteredApplications("ConvectionDiffusionApplication")
+
+# Import applications
+import KratosMultiphysics.ConvectionDiffusionApplication as ConvectionDiffusionApplication
+
+# Import base class file
+import convection_diffusion_base_solver
+
+def CreateSolver(main_model_part, custom_settings):
+    return ConvectionDiffusionTransientSolver(main_model_part, custom_settings)
+
+class ConvectionDiffusionTransientSolver(convection_diffusion_base_solver.ConvectionDiffusionBaseSolver):
+    """The transient class for convection-diffusion solvers.
+
+    Public member variables:
+    transient_settings -- settings for the implicit dynamic solvers.
+
+    See convection_diffusion_base_solver.py for more information.
+    """
+    def __init__(self, main_model_part, custom_settings):
+        # Set defaults and validate custom settings.
+        self.transient_settings = KratosMultiphysics.Parameters("""
+        {
+            "transient_parameters" : {
+                "dynamic_tau": 1.0,
+                "theta"    : 1.0
```

Comment:
Why is Backward Euler the default? Crank-Nicolson (0.5) is second order...

Reply:
Well, it gave me less problems

Reply:
I am not an expert

Reply:
Done